app/views/partials/personal_key/_key.html.erb

```diff
@@ -12,14 +12,16 @@
           date: content_tag(:strong, I18n.l(Time.zone.today, format: '%B %d, %Y')),
         ) %>
   </p>
-  <%= render ClipboardButtonComponent.new(clipboard_text: code, unstyled: true) %>
-  <%= render ClickObserverComponent.new(event_name: 'IdV: download personal key') do %>
-    <%= render DownloadButtonComponent.new(
-          file_data: code,
-          file_name: 'personal_key.txt',
-          unstyled: true,
-          class: 'margin-x-2 display-inline-block',
-        ).with_content(t('forms.personal_key.download')) %>
+  <% if show_download_buttons %>
```

Comment:
"download" could be a bit misleading since Download is just one of the three options to save the key. Maybe we don't qualify it (e.g. `show_buttons`) or qualify it more generically (e.g. `show_save_buttons`)?